stdlib/REPL/src/docview.jl

```diff
@@ -20,12 +20,12 @@ using Unicode: normalize
 ## Help mode ##
 
 # This is split into helpmode and _helpmode to easier unittest _helpmode
-helpmode(io::IO, line::AbstractString) = :($REPL.insert_hlines($io, $(REPL._helpmode(io, line))))
-helpmode(line::AbstractString) = helpmode(stdout, line)
+helpmode(io::IO, line::AbstractString, mod::Module) = :($REPL.insert_hlines($io, $(REPL._helpmode(io, line, mod))))
```

Comment:
The tests in REPL calls these functions explicitly so either these need to be given default arguments for `mod` or just change the tests.

Reply:
Oh right thank you, I had only run the repl.jl tests locally.